java-example/README.md

```diff
@@ -0,0 +1,91 @@
+# Auto Instrumenting Java Apps with OpenTelemetry
```

Comment:
Can you merge this commit with the previous one where you removed README.MD? If you do that, git will show that you just renamed the file without changing any of the content.